pkg/vmimage/data/validate.sh

```diff
@@ -20,17 +20,19 @@ EOF
 
 cp data/etc/yum.repos.d/kickstart.repo /etc/yum.repos.d/kickstart.repo
 
-yum check-update > /tmp/check || true
-yum updateinfo > /tmp/info || true
+yum check-update > /tmp/yum_check_update || true
+yum updateinfo > /tmp/yum_update_info || true
 
 
 # install openscap and run
 yum install -y openscap-scanner openscap-utils scap-security-guide
 
 oscap xccdf eval \
-  --profile stig-rhel7-disa \
+  --profile xccdf_cloud.osadev_profile_stig_customized_aro \
   --results /tmp/scap-results.xml \
   --report /tmp/scap-report.html \
+  --tailoring-file /root/data/ssg-rhel7-ds-aro.xml \
   --oval-results --fetch-remote-resources \
   --cpe /usr/share/xml/scap/ssg/content/ssg-rhel7-cpe-dictionary.xml \
-  /usr/share/xml/scap/ssg/content/ssg-rhel7-xccdf.xml 2>&1 > /tmp/oscap.log || true
+  /usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml > /tmp/oscap.log || true
```

Comment:
Is the plan to drop the `|| true` at some point in the future so that this check will fail if we fail the SCAP benchmark, right?

Reply:
@ehashman, Yes.  This will be a follow up when we hammer down the details regarding report storage or notifications.